### PR TITLE
Feat/footer

### PR DESCRIPTION
**Pull request by aiskairr**
Issue #:
[Trello](https://trello.com/c/aXSGh5Y6/5-footer-adaptive)

## PR type

Please check the type of change your PR introduces:

- [x] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe):

#### before:
Снимок экрана 2022-10-08 211023

#### after:
Снимок экрана 2022-10-08 211108

## Does this introduce a breaking change?

- [x] Yes (please describe):
- [ ] No

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Accessibility support has been tested
- [ ] Responsive layout and functionality have been tested
  - [x] desktop and mobile view responsiveness are supported
  - [ ] no loss of functionality or content when zoomed in to 400%
  - [x] no horizontal scrolling when zoomed in to 400% (only vertical scrolling, exception: DataGrids)
- [ ] avoids using many levels of prop drilling
- [x] check console on errors and warnings.
---

References:
- [Clean code](https://github.com/airbnb/javascript)

### CODE REVIEW

**Comment by Bezymeys** [comment]
please add screenshot

File: src/components/footer/Footer.jsx [hunk 1]
```diff
@@ -2,83 +2,73 @@ import { FooterLinks } from "../../constants/FooterLinks";
...
+            </div>
+            <div className={css.downloader_images}>
+              <img src="./images/footer/appstorebadgeenwhite-2x.png" alt="apple" />
+              <img src="./images/footer/image.png" alt="google" />
+            </div>
+            <div className={css.contacts}>
+              <img src="./images/footer/z-logo-default.svg" alt="z-logo" />
+              <div className={css.webImages}>
+                <p>{t("footer.followUs")}</p>
+                <img src="./images/footer/Facebook_Logo_(2019).png" alt="" />
```
Comment (Bezymeys) [inline src/components/footer/Footer.jsx:77]: wrap them link

File: src/components/footer/Footer.jsx [hunk 2]
```diff
@@ -2,83 +2,73 @@ import { FooterLinks } from "../../constants/FooterLinks";
...
+              <p>{t("footer.DRE")}</p>
+            </div>
+            <div>
+              <p>{t("footer.ContactZillow")}</p>
+            </div>
+            <div>
+              <p>{t("footer.listings")}</p>
+            </div>
+            <div className={css.downloader_images}>
+              <img src="./images/footer/appstorebadgeenwhite-2x.png" alt="apple" />
```
Comment (Bezymeys) [inline src/components/footer/Footer.jsx]: same & add gap between img

File: src/components/footer/Footer.jsx [hunk 3]
```diff
@@ -2,83 +2,73 @@ import { FooterLinks } from "../../constants/FooterLinks";
...
+              <img src="./images/footer/image.png" alt="google" />
+            </div>
+            <div className={css.contacts}>
+              <img src="./images/footer/z-logo-default.svg" alt="z-logo" />
```
Comment (Bezymeys) [inline src/components/footer/Footer.jsx:69]: same

File: src/components/footer/Footer.jsx [hunk 4]
```diff
@@ -2,83 +2,73 @@ import { FooterLinks } from "../../constants/FooterLinks";
...
+                <Link to="#">{t("footer.fair")}</Link>
+              </p>
+              <p>
+                {t("footer.TREC")}
+                <Link to="#">{t("footer.infoAbout")}</Link>
+              </p>
+              <p>{t("footer.DRE")}</p>
+            </div>
+            <div>
+              <p>{t("footer.ContactZillow")}</p>
```
Comment (Bezymeys) [inline src/components/footer/Footer.jsx]: change to contaсtZillow with small (C)
wrap to link

**Comment by Bezymeys** [comment]
please make gaps smaller

**Review by Bezymeys — CHANGES_REQUESTED** [review]

File: src/components/footer/Footer.jsx [hunk 5]
```diff
@@ -2,83 +2,73 @@ import { FooterLinks } from "../../constants/FooterLinks";
...
+    <>
+      <footer>
+        <div className={css.container}>
+          <div className={css.FooterLinks}>
+            <div className={css.navigateLinks}>
+              {FooterLinks.map((items) => (
+                <LinksCard key={items.id} {...items} />
+              ))}
+            </div>
+            <Link to="#">{t("footer.informatio")}</Link>
```
Comment (Bezymeys) [inline src/components/footer/Footer.jsx]: change style link

File: src/components/footer/Footer.jsx [hunk 6]
```diff
@@ -2,83 +2,73 @@ import { FooterLinks } from "../../constants/FooterLinks";
...
-          <div className={css.downloader_images}>
-            <img src="./images/footer/appstorebadgeenwhite-2x.png" alt="apple" />
-            <img src="./images/footer/image.png" alt="google" />
+    <>
+      <footer>
+        <div className={css.container}>
+          <div className={css.FooterLinks}>
+            <div className={css.navigateLinks}>
+              {FooterLinks.map((items) => (
+                <LinksCard key={items.id} {...items} />
```
Comment (Bezymeys) [inline src/components/footer/Footer.jsx]: эта карточка для линков обязателен?

**Review by Bezymeys — CHANGES_REQUESTED** [review]

**Comment by Bezymeys** [comment]
добавь контейнер в App.scss и задай ему ширину в px

File: src/components/footer/Footer.jsx (hunk 6 above)
Comment (aiskairr) [reply src/components/footer/Footer.jsx]: Да

**Review by Bezymeys — APPROVED** [review]

**Pull request merged**